x-pack/test/case_api_integration/basic/tests/cases/comments/find_comments.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+
+import { CASES_URL } from '../../../../../../plugins/case/common/constants';
+import { postCaseReq, postCommentReq } from '../../../../common/lib/mock';
+import { deleteCases, deleteCasesUserActions, deleteComments } from '../../../../common/lib/utils';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('legacyEs');
+
+  describe('find_comments', () => {
+    afterEach(async () => {
+      await deleteCases(es);
+      await deleteComments(es);
+      await deleteCasesUserActions(es);
+    });
+
+    it('should get all case comment', async () => {
```

Comment:
On `x-pack/plugins/case/server/routes/api/cases/comments/find_comments.ts` we have the following logic

```
 const args = query
          ? {
              client,
              caseId: request.params.case_id,
              options: {
                ...query,
                sortField: 'created_at',
              },
            }
          : {
              client,
              caseId: request.params.case_id,
            };
```

Do you think we should test that?